week-3/todo-list/script.js

```diff
@@ -1,25 +1,64 @@
-function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
-}
+let inputField = document.getElementById("input-field")
+let buttonAddTodo = document.getElementById("add-todo")
+let buttonDeleteLastTodo = document.getElementById("remove-last-todo")
+let buttonDeleteAllCompleted = document.getElementById("remove-all-completed")
+let list = document.getElementById("todo-list");
+
+
+buttonAddTodo.addEventListener("click", populateTodoList)
 
-// These are the same todos that currently display in the HTML
-// You will want to remove the ones in the current HTML after you have created them using JavaScript
-let todos = [
-  { task: "Wash the dishes", completed: false },
-  { task: "Do the shopping", completed: false },
-];
 
-populateTodoList(todos);
+function populateTodoList(event) {
+
+  let inputFieldValue = inputField.value;
 
-// This function will take the value of the input field and add it as a new todo to the bottom of the todo list. These new todos will need the completed and delete buttons adding like normal.
```

Comment:
This "complete" and "delete" buttons for each todo are missing from the current implementation